rhemyn/README.md

```diff
@@ -1,11 +1,11 @@
 # Rhemyn
-A basic experimental WIP regex engine/AOT Wasm compiler in JS. Regex engine for Porffor! Uses own regex parser, no dependencies (excluding porffor internals). <br>
+A basic experimental WIP regex engine/AOT Wasm compiler in JS. Regex engine for Porffor! Uses its own regex parser, no dependencies (excluding Porffor internals). <br>
 Age: ~1 day (of work)
 
 Made for use with Porffor but could possibly be adapted, implementation/library notes:
 - Exposes functions for each regex "operation" (eg test, match)
 - Given a regex pattern string (eg `a+`), it returns a "function" object
-- Wasm function returned expects an i32 pointer to a UTF-16 string (can add UTF-8 option later if someone else actually wants to use this)
```

Comment:
This actually isn't true anymore, could you just remove this line? (sorry :sweat_smile:)